sec-keys/openpgp-keys-danielkiper/openpgp-keys-danielkiper-20230215.ebuild

```diff
@@ -0,0 +1,26 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DESCRIPTION="OpenPGP keys used by Daniel Kiper"
+# I can't find a home page for him, so I'm just gonna put GRUB's since that's
+# currently his baby.
+# - xxc3nsoredxx
+HOMEPAGE="https://www.gnu.org/software/grub/"
+SRC_URI="
+	https://openpgpkey.net-space.pl/.well-known/openpgpkey/net-space.pl/hu/9b7hhehhfsjjw6h7bfzdesyjrhqhwgaq?l=dkiper
+		-> ${P}.asc
```

Comment:
The file is not ascii-armored, so ".asc" is probably not appropriate. Maybe use ".key" instead.

Reply:
Gotcha.

Reply:
I found a `.gpg` in my `/usr/share/openpgp-keys` so I think I'll use that. Even though some non-ASCII ones use `.asc`, I agree that it's not the best name to use.

I had blindly assumed it'd download an ASCII armored version since that's the most common way to share keys in my experience.